test/comment.js

```diff
@@ -100,7 +217,7 @@ describe('Parser → Comment', () => {
 
   fixtures.forEach((fixture) => {
     it(fixture.it, () => {
-      let ast = new Parser(fixture.test).parse(),
+      let ast = new Parser(fixture.test, { loose: fixture.loose }).parse(),
```

Comment:
so this is the only thing that stands out as questionable. the existing tests should continue to run without `loose`, the new tests should use `loose`, but they should be separate. we don't want non-loose tests running under `loose`.

Reply:
@shellscape i don't change any existing comments just add flag to run own comments in `loose` mode as you can do in other tests with `loose`

Reply:
yeah disregard that, i'm juggling too much at the moment.